features/dump.feature

```diff
@@ -32,3 +37,22 @@ Feature: The dump command
       Util\.system! scp -C user@www\.example\.com:~\/dumps\/dump_for_download.dump .*?\/tmp\/aruba\/tmp\/staging.dump
       """
       And the output should contain "> Dumped the staging database to tmp/staging.dump"
+
+  Scenario: Creating a dump of one of multiple remote databases and loading it locally
+    Given a file named "Capfile" with "Capfile exists"
+    And a file named "config/deploy.rb" with:
+    """
+    """
+    And a file named "config/deploy/staging.rb" with:
+    """
+    set :rails_env, 'staging'
+    set :deploy_to, '/var/www/example.com'
+    set :user, 'user'
+
+    server 'www.example.com'
+    """
+
+    When I run `geordi dump staging --database primary`
+    Then the output should contain "# Dumping the primary database of staging"
+      And the output should contain "Util.system! ssh, user@www.example.com, -t, cd /var/www/example.com/current && bash --login -c 'dumple staging primary --for_download'"
+      And the output should contain "> Dumped the primary staging database to tmp/staging.dump"
```

Comment:
I am missing the `--load` option in L55 and the tests for the expected loading behavior (as discussed):

```
      # Loading the dump
      And the output should contain "Sourcing dump into the development db"
      And the output should contain "Your development database has now the data of staging (primary database)."
```

Reply:
We realized that there was no feature test for the load option in general, so we implemented that as well.